tests/test_options.py

```diff
@@ -216,6 +216,16 @@ def cmd(username):
     assert "(current user)" in result.output
 
 
+def test_intrange_default_help_text(runner):
```

Comment:
Simple test case added to check for correct output from IntRange type parameters

Reply:
Looks good :+1: